src/app/api/updateTask/[id]/route.ts

```diff
@@ -0,0 +1,39 @@
+import prisma from "@/lib/prisma";
+import { z } from "zod";
+
+const ZTaskSchema = z.object({
+  id: z.number().optional(),
```

Comment:
**Question: Can you just grab the id from the params.route do you need it  here in this schema?**

Reply:
done